tests/unit/action-test.js

```diff
@@ -0,0 +1,50 @@
+import Ember from 'ember';
+import action from 'ember-computed-decorators/action';
+import { module, test } from 'qunit';
+
+module('Actions');
+
+test('action decorator works with traditional Ember.Object', function(assert) {
+  assert.expect(2);
+
+  const obj = Ember.Object.extend({
+    @action
+    foo(val) {
+      if (val === 'bar') {
+        assert.ok(true, 'action called through send');
+      } else if (val === 'baz') {
+        assert.ok(true, 'action called directly');
+      }
+    },
+
+    send(fn, ...args) {
```

Comment:
I'd say the easiest would be to make a "simple" component integration test instead.

```js
moduleForComponent('actions', { integration: true });

test('action decorator', function(assert) {
  this.register('component:foo-bar', class extends Ember.Component {
    @action
    foo() {
      assert.ok(true, 'called!');
    }
  });
  this.register('template:components/foo-bar', hbs`<button {{action 'foo'}}>Click Me!</button>`);

  this.render(hbs`{{foo-bar}}`);

  return click('button');
});
```